hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileCreateTest.java

```diff
@@ -122,7 +123,8 @@ private TransactionBody newCreateTxn(KeyList keys, long expirationTime) {
     void setUp() {
         subject = new FileCreateHandler();
         fileStore = new WritableFileStore(writableStates);
-        config = new FilesConfig(101L, 121L, 112L, 111L, 122L, 102L, 123L, 1000000L, 1024, 150L);
+        config = new FilesConfig(
+                101L, 121L, 112L, 111L, 122L, 102L, new LongPair(150L, 159L), 123L, 1000000L, 1024, 150L);
```

Comment:
I believe you could be using HederaTestConfigBuilder to avoid having to fill all these in...

Reply:
needed in order not to use the default values